osquery/tables/applications/posix/prometheus_metrics.h

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#pragma once
+
+#include <chrono>
+#include <map>
+#include <string>
+#include <vector>
+
+#include <osquery/tables.h>
+
+#include <boost/network/protocol/http/client.hpp>
+
+namespace osquery {
+namespace tables {
+
+const std::string kcol_target_name = "target_name";
```

Comment:
So i'm bit confused, i checked with @PoppySeedPlehzr about the const names last night before changing.   I see in the osquery source, there's both snake cased and camel cased names...